specification/components/examples/requests/NHAIS/removal.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
is no operation definition needed? e.g: delete?

Reply:
My understanding is that the URI defines the operation type e.g. /fhir/Patient/12345/$nhais.removal would perform an NHAIS removal operation on Patient id 12345 with parameters supplied in the payload